src/Group_User.php

```diff
@@ -385,8 +384,7 @@ public static function getDataForGroup(
         &$members,
         &$ids,
         $crit = '',
-        $tree = 0,
-        bool $check_entities = true
```

Comment:
We can't remove a property in a bugfix release, this would be a breaking chance.

Reply:
> We can't remove a property in a bugfix release, this would be a breaking change.

:+1: